packages/xstate-inspect/src/index.ts

```diff
@@ -68,80 +56,109 @@ export function createDevTools() {
   };
 }
 
-export const inspectMachine = createMachine<{
-  client?: any;
-}>({
-  initial: 'pendingConnection',
-  context: {
-    client: undefined
-  },
-  states: {
-    pendingConnection: {},
-    connected: {
-      on: {
-        'service.state': {
-          actions: (ctx, e) => ctx.client!.send(e)
-        },
-        'service.event': {
-          actions: (ctx, e) => ctx.client!.send(e)
-        },
-        'service.register': {
-          actions: (ctx, e) => ctx.client!.send(e)
-        },
-        'service.stop': {
-          actions: (ctx, e) => ctx.client!.send(e)
-        },
-        'xstate.event': {
-          actions: (_, e) => {
-            const { event } = e;
-            const scxmlEventObject = JSON.parse(event) as SCXML.Event<any>;
-            const service = serviceMap.get(scxmlEventObject.origin!);
-            service?.send(JSON.parse(event));
-          }
-        },
-        unload: {
-          actions: (ctx) => {
-            ctx.client!.send({ type: 'xstate.disconnect' });
-          }
-        },
-        disconnect: 'pendingConnection'
+export const createInspectMachine = (
+  devTools: XStateDevInterface = globalThis.__xstate__
```

Comment:
note: `globalThis` might not be available in all envs but I guess for the inspect package this might be OK

Reply:
Good call; for those other environments, the expectation is for developers to pass in a custom reference to `devTools` instead of this one (I'll add this to the docs once I get the Node + inspect implementation working)